cmake/app/boilerplate.cmake

```diff
@@ -14,7 +14,7 @@
 # modified by the entry point ${APPLICATION_SOURCE_DIR}/CMakeLists.txt
 # that was specified when cmake was called.
 
-# CMake version 3.8.2 is the real minimum supported version.
+# CMake version 3.13.1 is the real minimum supported version.
```

Comment:
I guess this comment is not really true with this version anymore?

Reply:
I can't see anything untrue here.

Reply:
Ok.. I read that sentence as meaning that we use functionality only supported in 3.13.1.
So that, for ex, if a user has, say 3.12, in his system and for his app, he set the minimum requirement to 3.12, He should expect things to not work.

If the sentence read: "Zephyr is only tested with cmake 3.13.1 or newer" the meaning I would understand would be another thing.

Reply:
> I read that sentence as meaning that we use functionality only supported in 3.13.1.

This will be true shortly after this is merged.

Reply:
Actually, because of this line:

https://github.com/zephyrproject-rtos/zephyr/pull/12022/files/98835cd8e738c1e42e8ce34f18cfba8ba6f888fd#diff-b9537eb0f71fef9b0fdb1e934a042cecR26

it is true immediately.